src/Symfony/Component/Console/Application.php

```diff
@@ -335,6 +335,8 @@ public function register($name)
     /**
      * Adds an array of command objects.
      *
+     * If a Command is not enabled it will not be added.
```

Comment:
sorry @Ener-Getick , I tried but it seems `@see` is no longer used within the SF community;
https://github.com/symfony/symfony/pull/19389#discussion_r72097495